Modules/CalcPerform.lua

```diff
@@ -720,6 +720,18 @@ function calcs.perform(env)
 		t_insert(extraAuraModList, value.mod)
 	end
 
+	-- Calculate number of active heralds
+	local heraldList = { }
+	for _, activeSkill in ipairs(env.player.activeSkillList) do
+		if activeSkill.skillCfg.skillName:match("^Herald of") then
```

Comment:
Is there any property of the skill we could use here that isnt string matching?

Reply:
I can match by `SkillType.Herald` but the indexing still has to be by name so I can count them